bin/rename_fasta.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+'''Rename contigs of a FASTA file with incremental count.'''
+
+import argparse
+
+
+def main():
+    '''Execute renaming.'''
+
+    # Parse arguments.
+    parser = argparse.ArgumentParser(description='Rename FASTA files.', epilog='Work out those contigs.')
```

Comment:
I think it's better to reuse the script's docstring here to reduce redundancy and risk of the two diverging in potential future updates to the code.

```suggestion
    parser = argparse.ArgumentParser(description=__doc__, epilog='Work out those contigs.')
```